lib/generators/rails_template18f/github_actions/templates/github/workflows/deploy-production.yml.tt

```diff
@@ -42,14 +42,12 @@ jobs:
         run: terraform apply -auto-approve -input=false
       <% end %>
       - name: Deploy app
-        uses: 18F/cg-deploy-action@main
+        uses: cloud-gov/cg-cli-tools@main
         env:
           RAILS_MASTER_KEY: ${{ secrets.RAILS_MASTER_KEY }}
         with:
           cf_username: ${{ secrets.CF_USERNAME }}
           cf_password: ${{ secrets.CF_PASSWORD }}
           cf_org: <%= cloud_gov_organization %>
           cf_space: <%= cloud_gov_production_space %>
-          push_arguments: >-
-            --vars-file config/deployment/production.yml
-            --var rails_master_key=$RAILS_MASTER_KEY
+          cf_command: push -vars-file config/deployment/production.yml --var rails_master_key=${{ env.RAILS_MASTER_KEY }} --strategy rolling
```

Comment:
[question] when is `${{ env.RAILS_MASTER_KEY }}` interpolated? Would it be possible/cleaner to set this to `${{ secrets.RAILS_MASTER_KEY }}` and remove the `env` block on lines 46-47?

Reply:
It's resolved in the course of running the workflow, so before the value reaches the action itself. No idea if it is redundantly set in `secrets`... Back when I made this PR I was just doing the minimum to get into the supported action!